modules/apps/portal-store/portal-store-gcs/build.gradle

```diff
@@ -43,4 +43,6 @@ dependencies {
 	compileOnly project(":apps:static:portal-configuration:portal-configuration-metatype-api")
 	compileOnly project(":core:petra:petra-io")
 	compileOnly project(":core:petra:petra-string")
+
+	testCompile group: "com.liferay.portal", name: "com.liferay.util.java", version: "default"
```

Comment:
@brandizzi Why this is needed?

Reply:
@tinatian it is not needed anymore. Some previous versions of this PR needed that, but we applied requested changes and this dependency is not necessary anymore. It was removed in this other PR: https://github.com/liferay-core-infra/liferay-portal/pull/2334